src/metadata.jl

```diff
@@ -1,69 +1,167 @@
-##############################################################################
-#
-# Metadata field getters and setters
-#
-##############################################################################
-
 import Languages.name
 
+"""
+    title(doc)
+
+Return the title metadata for `doc`.
+"""
```

Comment:
The docs encourage to write a short list of `See Also`, should I iterate through this PR once again and add those?
What do you say @laschuet @aquatiko 

For eg, in this case, we can add See Also list as `title!`, `titles`, `titles!`.